src/main/java/com/yahoo/bullet/rest/AsyncConfiguration.java

```diff
@@ -0,0 +1,96 @@
+/*
+ *  Copyright 2019, Yahoo Inc.
+ *  Licensed under the terms of the Apache License, Version 2.0.
+ *  See the LICENSE file associated with the project for terms.
+ */
+package com.yahoo.bullet.rest;
+
+import com.yahoo.bullet.common.BulletConfig;
+import com.yahoo.bullet.pubsub.PubSubResponder;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Objects;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+
+@Configuration @Slf4j
+@ConditionalOnProperty(prefix = "bullet.async", name = "enable", havingValue = "true")
+public class AsyncConfiguration {
+    @NoArgsConstructor @AllArgsConstructor @Slf4j
+    public static class ResponderClasses {
+        @Getter @Setter
+        private List<String> classes = new ArrayList<>();
+        private static final String CONFIG_KEY = "bullet.async.configuration.class.name";
+
+        /**
+         * Creates a {@link List} of {@link PubSubResponder} instantiated and configured classes out of the {@link List}
+         * of String class names.
+         *
+         * @param config The {@link BulletConfig} to instantiate the classes with. It should contain the superset of
+         *               all required configs needed by all the responder classes.
+         * @return The list of instantiated classes in the order provided.
+         */
+        List<PubSubResponder> create(BulletConfig config) {
+            Objects.requireNonNull(config);
+            Function<String, PubSubResponder> creator = createClass(config);
+            return classes.stream().map(creator).collect(Collectors.toList());
```

Comment:
This is a little unusual the way this CONFIG_KEY is set in the BulletConfig just so the `loadConfiguredClass()` function can be used.   

I know it's kind of late to make changes to bullet-core, but maybe it would be a little cleaner if we added a static function to Utilities.java class in bullet-core.  Something like `Utilities.loadClass(String className)` that just loaded the class with the given className.  

Then that could just be used directly like ```classes.stream().map(Utilities::loadClass).collect(Collectors.toList())```.  And that function could also be used in `BulletConfig.loadConfiguredClass()`?

Or if that's too complicated to change bullet-core again, maybe just add a comment above clarifying that the `bullet.async.configuration.class.name` setting should never be set in a BulletConfig file - and only used internally by this class?

Reply:
I added a static method to this [PR](https://github.com/bullet-db/bullet-core/pull/79) and will use that after